### ISSUE TITLE
Question about Watchers

### ISSUE THREAD

**Issue by SquareShark** [elastic/terraform-provider-elasticstack#337]
Hi, Loving the new version update. Thank you for all of your hard work!. I have a use case where I configure painless scripts with my watchers in the Transform block. Ref: https://www.elastic.co/guide/en/elasticsearch/reference/8.8/transform-script.html

 Upon reading through the docs related to the Watcher API and looking at the watch.go file, I could not find anything that would suggest I can have this Transform block in my tfplan, is there a way to add this in? (sorry for this is a silly question, I am very new to this). If its not possible is it okay if I was to request this functionality pretty please?.

Looking to add in something similar to the example found on this docs page: https://www.elastic.co/guide/en/elasticsearch/painless/current/painless-watcher-transform-context.html
```
POST _watcher/watch/_execute
{
  "watch" : {
    "trigger" : { "schedule" : { "interval" : "24h" } },
    "input" : {
      "search" : {
        "request" : {
          "indices" : [ "seats" ],
          "body" : {
            "query" : { "term": { "sold": "true"} },
            "aggs" : {
              "theatres" : {
                "terms" : { "field" : "play" },
                "aggs" : {
                  "money" : {
                    "sum": { "field" : "cost" }
                  }
                }
              }
            }
          }
        }
      }
    },
    "transform" : {
      "script":
      """
        return [
          'money_makers': ctx.payload.aggregations.theatres.buckets.stream()  
            .filter(t -> {                                                    
                return t.money.value > 50000
            })
            .map(t -> {                                                       
                return ['play': t.key, 'total_value': t.money.value ]
            }).collect(Collectors.toList()),                                  
          'duds' : ctx.payload.aggregations.theatres.buckets.stream()         
            .filter(t -> {
                return t.money.value < 15000
            })
            .map(t -> {
                return ['play': t.key, 'total_value': t.money.value ]
            }).collect(Collectors.toList())
          ]
      """
    },
    "actions" : {
      "my_log" : {
        "logging" : {
          "text" : "The output of the payload was transformed to {{ctx.payload}}"
        }
      }
    }
  }
}
```

**Comment by RobsonSutton** [comment]
Hi @SquareShark 👋

Great spot and apologies about missing this when I added watchers the other month! 
It appears that there is a documentation error surrounding the API's (it makes no mention of the transform field in the API docs despite it being supported! - https://www.elastic.co/guide/en/elasticsearch/reference/current/watcher-api-put-watch.html)

I have just raised a PR which will add this field in so that the full functionality is supported, I'll also look to just fix that documentation issue on the API doc end of things too 👍

**Comment by tobio** [comment]
This PR has been released in 0.6.1
